usr/bin/sandbox-app-launcher

```diff
@@ -7,8 +7,9 @@ set -e
 
 app_name="${1}"
 shift 1
+: ${app_path:="$(type -P "${app_name}" || true)"}
 
-if ! [[ "${app_name}" =~ [0-9a-zA-Z/] ]]; then
+if ! [[ "${app_name}" =~ [0-9a-zA-Z/] ]] || ! [[ "${app_path}" =~ [0-9a-zA-Z/] ]]; then
```

Comment:
This is a bit difficult to read. Could you please move that into a separate `if`?